src/contour/opengl/TerminalWidget.cpp

```diff
@@ -105,6 +105,8 @@ using std::chrono::steady_clock;
 
 using actions::Action;
 
+#define CONTOUR_INIT_RENDERER_EARLY 1
```

Comment:
Hardcoded macro? Do you intend this to become a build-time (CMake) configuration parameter?

Reply:
Hardcoded because I wasn't sure if that's the root cause. With your confirmation I'll be refactoring it such that it'll be always initialized early (I think).

Reply:
> Do you intend this to become a build-time (CMake) configuration parameter?
I still have to confirm with (especially) Mac not being broken by this change. In worst case it'll become an `#if __APPLE__` rather a cmake option. but I hope I don't need to.